### DISCUSSION TITLE
how stop a recursive sending

### DISCUSSION THREAD

**Discussion by DoctorSubtilis** [Extended-Thunder/send-later#735] [General]
When I have set to re-send periodically an e-mail, but I want revoce this command, how should I do to stop the re-sending process?

**Comment by jikamens** [comment]
1. Consult the [user guide](https://extended-thunder.github.io/send-later/#what-happens-when-you-schedule-a-message).
2. Remove the scheduled message from your Drafts folder, or just start editing it, which turns it back into a regular, unscheduled draft.

**Reply by DoctorSubtilis** [reply]
ok, thank you!
